### PR TITLE
[docs] Add wave 2b link, lock 10000.1 cGLD

### PR DESCRIPTION
**Pull request by timmoreton**
### Description

* Add wave 2b gist link to Celo Gold Holders quick start guide
* Increase amount of cGLD that validator setup guides lock to `100001e17` in order to avoid bug where rounding error can cause locked gold amounts of exactly `10000` to not deliver voter rewards

### CODE REVIEW

File: packages/docs/celo-gold-holder-guide/quick-start.md [hunk 1]
```diff
@@ -100,11 +100,13 @@ Next, you will find the address of the `ReleaseGold` contract deployed for your
...
+- [ReleaseGold deployments on 4/30/20 ("Wave 1b")](https://gist.githubusercontent.com/timmoreton/32d36dcc48e52a4747df8cfa8e4a4d0f/raw/1557209b59fa863cc8ed9b66e72e6111fbfe3475/CeloRC1ReleaseGoldWave1b.json)
 
-- [RC1 ReleaseGold deployments on 5/16/20 ("Wave 2")](https://gist.githubusercontent.com/timmoreton/27e975bbca63723e218288b1a1f9fa54/raw/d55f51fde470fe040407c02462aeb5a7adc3d57f/CeloRC1ReleaseGoldWave2.json): includes self-custodying CoinList auction participants that completed the address submission process by 5/14/20
+- [ReleaseGold deployments on 5/16/20 ("Wave 2")](https://gist.githubusercontent.com/timmoreton/27e975bbca63723e218288b1a1f9fa54/raw/d55f51fde470fe040407c02462aeb5a7adc3d57f/CeloRC1ReleaseGoldWave2.json): includes self-custodying CoinList auction participants that completed the address submission process by 5/14/20
+
+- [ReleaseGold deployments on 5/29/20 ("Wave 2b")](https://gist.githubusercontent.com/timmoreton/f22e70b8149706f738327da8ff71c7d7/raw/944458feda304c7077f2bbb3e190ec677a5ed976/CeloMainnetReleaseGoldWave2b.json): includes self-custodying CoinList auction participants that completed the address submission process by 5/16/20
```
Comment (pranaymohan) [inline packages/docs/celo-gold-holder-guide/quick-start.md]: should this be 5/19?

File: packages/docs/getting-started/running-a-validator-in-baklava.md [hunk 2]
```diff
@@ -352,11 +352,11 @@ celocli account:show $CELO_VALIDATOR_GROUP_RG_ADDRESS
 celocli account:show $CELO_VALIDATOR_RG_ADDRESS
 ```
 
-Lock cGLD from your `ReleaseGold` contracts to fulfill the lock-up requirements to register a Validator and Validator Group. The current requirement is 10,000 cGLD to register a Validator, and 10,000 cGLD _per member validator_ to register a Validator Group.
+Lock cGLD from your `ReleaseGold` contracts to fulfill the lock-up requirements to register a Validator and Validator Group. The current requirement is any value stricly greater than 10,000 cGLD to register a Validator, and any value stricly greater than 10,000 cGLD _per member validator_ to register a Validator Group. Here we lock up 10000.1 cGLD for each.
```
Comment (pranaymohan) [inline packages/docs/getting-started/running-a-validator-in-baklava.md]:
```suggestion
Lock cGLD from your `ReleaseGold` contracts to fulfill the lock-up requirements to register a Validator and Validator Group. The current requirement is any value strictly greater than 10,000 cGLD to register a Validator, and any value strictly greater than 10,000 cGLD _per member validator_ to register a Validator Group. Here we lock up 10000.1 cGLD for each.
```

File: packages/docs/getting-started/running-a-validator-in-mainnet.md [hunk 3]
```diff
@@ -396,11 +396,11 @@ celocli account:show $CELO_VALIDATOR_GROUP_RG_ADDRESS
 celocli account:show $CELO_VALIDATOR_RG_ADDRESS
 ```
 
-Lock cGLD from your `ReleaseGold` contracts to fulfill the lock-up requirements to register a Validator and Validator Group. The current requirement is 10,000 cGLD to register a Validator, and 10,000 cGLD _per member validator_ to register a Validator Group.
+Lock cGLD from your `ReleaseGold` contracts to fulfill the lock-up requirements to register a Validator and Validator Group. The current requirement is any value stricly greater than 10,000 cGLD to register a Validator, and any value stricly greater than 10,000 cGLD _per member validator_ to register a Validator Group. Here we lock up 10000.1 cGLD for each.
```
Comment (pranaymohan) [inline packages/docs/getting-started/running-a-validator-in-mainnet.md]:
```suggestion
Lock cGLD from your `ReleaseGold` contracts to fulfill the lock-up requirements to register a Validator and Validator Group. The current requirement is any value strictly greater than 10,000 cGLD to register a Validator, and any value strictly greater than 10,000 cGLD _per member validator_ to register a Validator Group. Here we lock up 10000.1 cGLD for each.
```

**Review by pranaymohan — COMMENTED** [review]
LGTM plus nits -- thank you!

File: packages/docs/celo-gold-holder-guide/quick-start.md (hunk 1 above)
Comment (timmoreton) [reply packages/docs/celo-gold-holder-guide/quick-start.md]: agh, thanks. fixed.

**Review by pranaymohan — APPROVED** [review]

**Pull request merged**